protoc-gen-openapiv2/options/openapiv2.proto

```diff
@@ -21,6 +21,28 @@ enum Scheme {
 //
 // See: https://github.com/OAI/OpenAPI-Specification/blob/3.0.0/versions/2.0.md#swaggerObject
 //
+// Example:
+//
+//  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
```

Comment:
```suggestion
//  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
```

Reply:
Yep, should have spotted that. All updated.